src/shared-components/FormView/BrokerProperties/ConfigurationPage/AcceptorsConfigPage/AcceptorsConfigPage.tsx

```diff
@@ -33,13 +34,25 @@ export const AcceptorsConfigPage: FC<AcceptorsConfigProps> = ({ brokerId }) => {
 
   const addNewConfig = () => {
     if (configType === ConfigType.acceptors) {
+      const newPort = getConfigPort(
+        cr,
+        configType,
+        `acceptor-${configs.length}`,
+      );
       dispatch({
         operation: ArtemisReducerOperations.addAcceptor,
+        payload: { name: `acceptor-${configs.length}`, port: newPort },
       });
     }
     if (configType === ConfigType.connectors) {
+      const newPort = getConfigPort(
+        cr,
+        configType,
+        `connector-${configs.length}`,
+      );
       dispatch({
         operation: ArtemisReducerOperations.addConnector,
+        payload: { name: `connector-${configs.length}`, port: newPort },
```

Comment:
See, we already can see some logic externalized from the reducer to the frontend code. A hint for you: the correct solution should imply 0 changes to the UI side of the code.